Resources/config/yui.yml

```diff
@@ -346,6 +348,17 @@ system:
                 ez-roleserversideviewservice:
                     requires: ['ez-serversideviewservice', 'parallel', 'ez-translator']
                     path: "%ez_platformui.public_dir%/js/views/services/ez-roleserversideviewservice.js"
+                ez-subitemboxviewservice:
+                    requires:
+                        - 'parallel'
```

Comment:
well it is, to wait the content search to finish before giving the location and content to the response